### PR TITLE
Implement acronym exercise

### PR DESCRIPTION
**Pull request by lilislilit**
Impemented first of the exercises mentioned in #37 .

### CODE REVIEW

File: exercises/acronym/acronym.ts [hunk 1]
```diff
@@ -0,0 +1,5 @@
+export default class Acronyms {
+  static parse(phrase: string): string {
```
[the remark this answers is not in the document]
Comment (lilislilit) [reply exercises/acronym/acronym.ts]: Are we sure that this bot is okay? Linter does not see any extra semicolons.

File: exercises/acronym/acronym.ts [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+    return '';
```
Comment (masters3d) [inline exercises/acronym/acronym.ts]: The bot was off by one line. :)

File: exercises/acronym/acronym.test.ts [hunk 3]
```diff
@@ -0,0 +1,27 @@
+import Acronyms from './acronym'
```
Comment (masters3d) [inline exercises/acronym/acronym.test.ts]: By convention this would be singular. Acronym.

**Comment by masters3d** [comment]
Thanks!

**Pull request merged**